spec/views/members/show.html.erb_spec.rb

```diff
@@ -12,4 +12,15 @@
     }
   end
 end
+
+RSpec.describe "members/show.html.erb", type: :view do
+	context 'it display the end_subscription' do
+		it "displays end_subscription" do
+			assign(:member, build(:member, end_subscription: Date.new(2020, 10, 10) ))
+      allow(view).to receive(:current_member) { build_stubbed :member }
+			render
+			expect(rendered).to match '10-10-2020' 
+		end
+	end
+end
```

Comment:
This information should not appear on members/show.html, but should instead appear in the admin interface.